Doesn't work out-of-the-box

**Bug**

When installed globally, this breaks @dojo/cli's `create` function.

Package Version: 0.1.0

**Code**

```
$ npm install -g @dojo/cli
$ npm install -g @dojo/cli-create-widget
$ dojo create
Commands are not available: Error: Failed to load module /usr/local/lib/node_modules/@dojo/cli-create-widget
Nested error: Cannot find module '@dojo/cli-create-app/createDir'
```

Manually installing `@dojo/cli-create-app` globally resolves the issue.

**Expected behavior:**

`dojo create` should work

**Actual behavior:**

An error is emitted.

--- a/package.json
+++ b/package.json
@@ -26,7 +26,6 @@
     "test": "grunt test"
   },
   "devDependencies": {
-    "@dojo/cli-create-app": "~0.3.0",
     "@dojo/core": "~0.3.0",
     "@dojo/has": "~0.1.1",
     "@dojo/interfaces": "~0.2.0",
@@ -52,6 +51,7 @@
     "yargs": "^5.0.0"
   },
   "dependencies": {
+    "@dojo/cli-create-app": "~0.3.0",
     "chalk": "^1.1.3",
     "cross-spawn": "^4.0.0",
     "ejs": "^2.5.2",

